packages/data/src/components/use-dispatch/test/use-dispatch.js

```diff
@@ -19,39 +28,39 @@ describe( 'useDispatch', () => {
 
 	it( 'returns dispatch function from store with no store name provided', () => {
```

Comment:
Here we could use a new `renderHook` feature, but it's only available in RTL v13.1.0, which also requires React 18.

Meanwhile, I'm using a similar method as `renderHook` to tests hook return value - https://github.com/eps1lon/react-testing-library/blob/v13.4.0/src/pure.js#L223-L233